charts/iter8/values.yaml

```diff
@@ -4,16 +4,20 @@ iter8Image: iter8/iter8:0.15
 ### majorMinor is the minor version of Iter8
 majorMinor: v0.15
 
-### runner for Kubernetes experiments may be job, cronjob, or none
+# TODO: Should this only ever be job?
+### runner for Kubernetes experiments may be job or none
 runner: none
```

Comment:
Extra attention here: does this need to be here? Is runner always assumed to be job?